silo/migrations/0040_auto_20180125_0149.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.3 on 2018-01-25 09:49
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('silo', '0039_auto_20171205_0847'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='read',
+            name='task_id',
+            field=models.CharField(blank=True, default=None, max_length=50, null=True, verbose_name=b'Celery task id'),
+        ),
+        migrations.AlterField(
```

Comment:
I'm wondering why this migration also alters 3 different fields which have nothing to do with adding task_id field... Weird. Maybe we should manually delete them?

Reply:
Good point. I didn't realise other migrations were pending. Taken out